image_fixtures.go

```diff
@@ -68,7 +68,8 @@ func skopeoCopyDockerArchiveToPath(t *testing.T, dockerArchivePath, destination
 		t.Fatalf("cannot find skopeo executable")
 	}
 
-	cmd := exec.Command("skopeo", "copy", fmt.Sprintf("docker-archive:%s", dockerArchivePath), destination)
+	archive := fmt.Sprintf("docker-archive:%s", dockerArchivePath)
+	cmd := exec.Command("skopeo", "copy", archive, destination)
```

Comment:
@wagoodman an extra set of eyes on this one that I just changed would be great

Reply:
looks good to me (I don't see why the former is a security risk, I'll do some reading)